apps/meteor/client/hooks/useFeaturePreviewEnableQuery.ts

```diff
@@ -0,0 +1,24 @@
+import type { FeaturePreviewProps } from '@rocket.chat/ui-client';
+
+export const useFeaturePreviewEnableQuery = (features: FeaturePreviewProps[]) => {
+	return Object.entries(
+		features
+			.map((item) => {
+				if (item.enableQuery) {
```

Comment:
nitpick:
```
if (!item.enableQuery) {
	return item
}
...
```

Reply:
I don't think this makes much of a difference, we are returning the item object either way, only applying  more logic in case enableQuery is present.

Reply:
Yep, this is not a change on semantics but on styling. 1 less level of nesting is always a good thing to do. The early return helps with it.

wdyt?

Reply:
This is done just to reduce the code kamehameha (reduces indentation thus facilitating reading).

In the end of the day it is the same thing, we either enforce this pattern as a rule or resist making this kind of review. Think of all time spent discussing basic indentation (or using switch vs if) that could be invested in more relevant improvements.

TL;DR: Early return better, time coding > time discussing indentation

Reply:
I still believe this won't make much of a difference... IMHO this code although nested is still simple to read. 

If the early return was beneficial for performance, I would agree with you, tho.

Reply:
We also have this on our guides: https://developer.rocket.chat/docs/code-style-guide

Reply:
Even better :). Now we don't have to document it.